dsalgo/doubly_linked_list.py

```diff
@@ -0,0 +1,50 @@
+class Node(object):
+    # Doubly linked node
```

Comment:
It is a comment written to show initialization of doubly linked node i.e.
    def __init__(self, data=None, next=None, prev=None):
        self.data = data
        self.next = next
        self.prev = prev